lib/trino-orc/src/main/java/io/trino/orc/StripeReader.java

```diff
@@ -237,17 +237,18 @@ public Stripe readStripe(StripeInformation stripe, AggregatedMemoryContext syste
     private static boolean isSupportedStreamType(Stream stream, OrcTypeKind orcTypeKind)
     {
         if (stream.getStreamKind() == BLOOM_FILTER) {
-            // non-utf8 bloom filters are not allowed for character types
-            // non-utf8 bloom filters are not supported for timestamp
             switch (orcTypeKind) {
                 case STRING:
                 case VARCHAR:
+                    // non-utf8 bloom filters are not allowed for character types
```

Comment:
one line lower

Reply:
and I would explicitly add `return false ` to `STRING` `VARCHAR` CHAR` group

Reply:
good point